release-notes/SRM/SRM_2.0.2.md

```diff
@@ -0,0 +1,89 @@
+- A warning when the directory contains over 25,000 files.
+- A warning when the directory exceeds 1 GB of storage.
+- An error when the directory contains over 100,000 files.
+- An error when the directory exceeds 5 GB of storage.
+
+## Enhancements
+
+### Changes
+
+#### CommitReservation method improved [ID 40064]
+
+The *CommitReservation* method has been updated so that it now always waits for an ongoing booking to be ongoing again after an update. This prevents possible issues with resource swapping for ongoing bookings.
+
+#### Number of SRM booking log files limited in log collector configuration [ID 40320]
+
+In the SLLogCollector configuration, a limit has been introduced of at most 2000 booking log files for SRM. This means that log packages will now only include the 2000 most recently updated SRM booking log files.
+
+#### SkipServiceHandling automatically enabled + SRMServiceInfo objects no longer used [ID 40666]
+
+The *SkipServiceHandling* option of Resource Manager, which is available from DataMiner 10.4.9/10.5.0 onwards<!-- RN 39939 -->, will now be automatically enabled during the startup of the Booking Manager element. This will improve the performance of the SRM framework.
+
+In addition, in order to no longer rely on *SRMServiceInfo* objects, which are no longer needed, the framework has been extended to no longer create or update those objects.
+
+> [!IMPORTANT]
+> In existing setups, you will need to run the *SRM_MigrateSrmServiceInfo* script to adjust the *SRMServiceInfo* objects.
+
+#### Maximum pre-roll and post-roll duration increased to 7 days [ID 40907]
+
+Previously, the pre-roll and post-roll duration was limited to a maximum of one day. Now, the Skyline Booking Manager connector supports a maximum value of 7 days instead. All default SRM wizards have also been updated accordingly.
+
+#### SRM_ImportFunctions improvements [ID 40964]
+
+The behavior of the *SRM_ImportFunctions* script has been adjusted as follows when protocol functions are imported:
+
+- When no functions exist for a protocol, the import will take the functions with the provided version.
+- When functions already exist for a protocol, the import will consider only functions that have a version superior to the existing version, if the function is different from the current function.
+- When functions already exist for a protocol, and the provided import does not have all the active functions, the ones from the import will be concatenated with the existing active functions.
+
+#### Skyline Booking Manager: Debug logging behavior adjusted for QAction 330 [ID 41118]
+
+In the Skyline Booking Manager connector, the debug logging behavior of QAction 330 has been adjusted. Previously, all information related to this QAction was logged when the log type was set to *Debug*. Now, logging will only be added for this QAction when both the log type is set to *Debug* and the log level is set to 1, preventing unnecessary log entries.
```

Comment:
The RN was update of the initial version.

With the new change, these debug logs have been removed, which reduces unnecessary log entries and improves overall system clarity when the log type is set to "Debug".